nimare/utils.py

```diff
@@ -386,3 +390,36 @@ def uk_to_us(text):
         pattern = re.compile(r"\b(" + "|".join(SPELL_DICT.keys()) + r")\b")
         text = pattern.sub(lambda x: SPELL_DICT[x.group()], text)
     return text
+
+
+def use_memmap(logger):
```

Comment:
Might want a unit test for `use_memmap`, but I can do that in a separate pull request if you do not want to that here.